helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml

```diff
@@ -49,7 +49,7 @@ spec:
         ovdcNetwork: {{ .Values.cloudDirector.ovdcNetwork | quote }}
         site: {{ .Values.cloudDirector.site | quote }}
         vAppName: {{ include "resource.default.name" $  | quote }}
-        vipSubnet: {{ .Values.network.loadBalancer.vipSubnet | quote }}
+        vipSubnet: {{ index .Values.connectivity.network.loadBalancers.cidrBlocks 0 | quote }}
```

Comment:
We allow multiple cidrBlocks here but only the first one is used here? What's the purpose?

Reply:
I just followed the same logic we use in other places. I am not sure whether it is possible to use dual-stack for loadbalancers too. @vxav is it possible?

Reply:
While I am all for unifying across providers, personally I am not a big fan of obfuscating those provider specific fields as they will not match the fields in the CRs and will make debugging, editing and such more complicated (I can see myself wondering what `cidrBlocks` is in a few months time). 

In terms of using an array, I don't think it is supported according to https://github.com/vmware/cluster-api-provider-cloud-director/blob/main/config/crd/bases/infrastructure.cluster.x-k8s.io_vcdclusters.yaml#L511

Reply:
I used `vipSubnet` again and it is a string now.